stringfuzz/smt.py

```diff
@@ -80,31 +80,31 @@ def smt_int_lit(value):
     return IntLitNode(value)
 
 def smt_assert(exp):
-    return ExpressionNode('assert', [exp])
+    return ExpressionNode(IdentifierNode('assert'), [exp])
 
 def smt_and(a, b):
-    return ExpressionNode('and', [a, b])
+    return ExpressionNode(IdentifierNode('and'), [a, b])
```

Comment:
why are these identifier nodes?

Reply:
Because according to the grammar, they actually are, since new functions can be defined. It was inconsistent to have some functions have symbols of type `str` and others of type `IdentifierNode`.

Reply:
does this also apply to get-info, set-info, assert, etc.?

Reply:
Yes. They're all parsed in as identifiers.

Reply:
Sounds good!